opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -683,14 +683,14 @@ def __exit__(
 
         super().__exit__(exc_type, exc_val, exc_tb)
 
-    def record_error(self, err: Exception) -> None:
-        """Records an error as a span event."""
+    def record_exception(self, exception: Exception) -> None:
```

Comment:
Maybe for another PR, but is passing in the type, message and stack trace as parameters also valid for this method?

Reply:
Based on the spec, the recommendation was to have a convenience method to record exceptions (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#record-exception). 

This seemed more pythonic than passing in exception parameters, but i'm happy to add another PR if you feel strongly about it.